app/controllers/books_controller.rb

```diff
@@ -3,8 +3,10 @@ class BooksController < ApplicationController
   before_action :set_book, only: [:show]
 
   def index
-    @q = Book.ransack(params[:q])
-    @books = @q.result(distinct: true).page(params[:page])
+    sort = params[:sort] || "published_date DESC"
+    sort = "order_count DESC" if sort == "most_popular"
```

Comment:
You could remove this line if you change [this line](https://github.com/smashingboxes/ruti-bookstore/pull/27/files#diff-b9cabdc15aa47777492ecb56db9bfac3R5) to be `sort: "order_count"`